crypto/s2n_cbc_cipher_3des.c

```diff
@@ -34,9 +34,9 @@ static int s2n_cbc_cipher_3des_encrypt(struct s2n_session_key *key, struct s2n_b
 
     GUARD_OSSL(EVP_EncryptInit_ex(key->evp_cipher_ctx, NULL, NULL, NULL, iv->data), S2N_ERR_KEY_INIT);
 
-    int len = out->size;
+    int len = (int)out->size;
```

Comment:
It might be helpful to have a checked cast macro for things like this.

Reply:
i was thinking something similar.  i have more to do on this PR, so will incorporate this too.